lib/simctl.js

```diff
@@ -281,7 +281,48 @@ async function getScreenshot (udid:string):string {
   return screenshotImg.toString('base64');
 }
 
+/**
+ * Set the content of Simulator pasteboard (xcode >= 8.1 only)
+ *
+ * @param {string} udid - Device UDID.
+ * @param {string} content - The actual string content to be set.
+ * @param {string} encoding - The encoding of the given pasteboard content. UTF-8 by default.
+ */
+async function setPasteboard (udid:string, content:string, encoding = 'utf-8'):void {
+  const pbCopySubprocess = new SubProcess('xcrun', ['simctl', 'pbcopy', udid]);
+  await pbCopySubprocess.start(0);
+  const exitCodeVerifier = pbCopySubprocess.join();
+  const stdin = pbCopySubprocess.proc.stdin;
+  stdin.setEncoding(encoding);
+  stdin.write(content);
+  stdin.end();
+  await exitCodeVerifier;
+}
+
+/**
+ * Get the content of Simulator pasteboard (xcode >= 8.1 only)
+ *
+ * @param {string} udid - Device UDID.
+ * @param {string} encoding - The encoding of the returned pasteboard content. UTF-8 by default.
+ *
+ * @return {string} Current content of Simulator pasteboard or an empty string.
+ */
+async function getPasteboard (udid:string, encoding = 'utf-8'):string {
+  const args = ['simctl', 'pbpaste', udid];
+  try {
+    const {stdout} = await exec('xcrun', args, {timeout: 0, encoding});
+    return stdout;
+  } catch (e) {
+    if (e.stderr) {
+      log.errorAndThrow(`Error running '${args.join(' ')}': ${e.stderr.trim()}`);
```

Comment:
fixed.

It would be nice to have Xcode 8.3 or better Xcode9 to be available for Travis testing for this project, because now my functional test is skipped.
BTW, these tests are executed by `gulp unit-tests` command, but they have nothing to do with unit testing. Is that expected?